src/tests/AddNoteForm.test.js

```diff
@@ -0,0 +1,24 @@
+import { render, fireEvent } from '@testing-library/react';
+import { GlobalContext } from '../context/GlobalContext';
+import { BrowserRouter } from 'react-router-dom';
+import AddNoteForm from '../components/AddNoteForm';
+
+describe('add note form', () => {
+  test('render add note form', () => {
+    const initialState = {};
+    const { getByRole } = render(
+      <GlobalContext.Provider value={initialState}>
+        <BrowserRouter>
+          <AddNoteForm />
+        </BrowserRouter>
+      </GlobalContext.Provider>
+    );
+    const title = getByRole('heading');
+    expect(title).toBeInTheDocument();
+    expect(title.textContent).toBe('Add a Note');
+    const noteInput = getByRole('textbox');
+    expect(noteInput).toBeInTheDocument();
+    fireEvent.change(noteInput, { target: { value: 'my note content' } });
+    expect(noteInput.value).toBe('my note content');
+  });
```

Comment:
I think a lot of the time (especially with enzyme I think, but don't quote me on this as I'm not sure 100%) the render tests are not the same as the one who actually tests a function or code block. 
In this case, I would suggest in the future dividing it into this:
- render
- adding the note (filling the "form")
These next one is extra that is worth adding 
- testing you can't add an empty note